packages/@uppy/core/src/index.js

```diff
@@ -1,7 +1,7 @@
 /* global AggregateError */
 const Translator = require('@uppy/utils/lib/Translator')
 const ee = require('namespace-emitter')
-const cuid = require('cuid')
+const { nanoid: cuid } = require('nanoid')
```

Comment:
Why not rename to `nanoid` altogether?

Reply:
Laziness mostly – I was able to make the change with a simple search/replace, but yeah, it should probably be fixed before landing.